app/controllers/pages_controller.rb

```diff
@@ -1,3 +1,6 @@
 class PagesController < ApplicationController
-  def welcome; end
+  def welcome
+    @transaction = Transaction.new
+    @transactions = current_user.transactions.limit(10).order(created_at: :desc) if user_signed_in?
```

Comment:
I decided to leave ```TransactionsController#index``` for future ```activity_page```, instead of to be a root path for form with money and 10 last transactions.